docs/sources/developers/http_api/access_control.md

```diff
@@ -418,15 +484,23 @@ Content-Type: application/json; charset=UTF-8
 }
 ```
 
+#### Update role validation errors
+
+Permission validation only takes place when permission validation is enabled (`rbac.permission_validation_enabled = true`).
+
+> It has been enabled by default since Grafana 10.2
+
+[Same as the create role validation errors.]({{< relref "#create-role-validation-errors" >}})
```

Comment:
I don't recommend linking an entire sentence. Write link text as follows:

For more information, refer to [Create role validation errors](#create-role-validation-errors).

Please note that you don't need a `relref` for anchors.

Reply:
Done and changed `relref` to `ref` ;)